src/secrets.js

```diff
@@ -26,18 +26,13 @@ function masterKey() {
 function encrypt(secrets) {
   const salt = crypto.randomBytes(AES_SALT_BYTES);
   const iv = crypto.randomBytes(AES_IV_BYTES);
-
-  // construct key
   const key = crypto.pbkdf2Sync(masterKey(), salt, PBKDF2_ROUNDS, PBKDF2_KEYLEN, PBKDF2_DIGEST);
-
   const cipher = crypto.createCipheriv(ALGORITHM, key, iv);
-  const cipherText = Buffer.concat([cipher.update(JSON.stringify(secrets), "utf8"), cipher.final()]);
+  const cipherText = Buffer.concat([cipher.update(JSON.stringify(secrets), TEXT_ENCODING), cipher.final()]);
   const authTag = cipher.getAuthTag();
   const combinedData = Buffer.concat([cipherText, authTag]);
 
-  return `${ENCODING_PREFIX}${PBKDF2_ROUNDS}-${PBKDF2_KEYLEN}-${salt.toString("base64")}-${iv.toString(
-    "base64"
-  )}-${combinedData.toString("base64")}`;
+  return `${ENCODING}:${PBKDF2_ROUNDS}:${salt.toString(ENCODING)}:${iv.toString(ENCODING)}:${combinedData.toString(ENCODING)}`;
```

Comment:
Can you add a test that ensures a known cipher text can be decrypted? That'll ensure we don't break backwards compatibility in the future. I'd expect that test to break on a PR like this.

Reply:
Test added at https://github.com/DopplerHQ/gitops-secrets-nodejs/blob/main/tests/secrets.test.js#L42